.github/workflows/release.yml

```diff
@@ -35,3 +35,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.SEEK_OSS_CI_NPM_TOKEN }}
+
+      - name: Publish Storybook
+        run: yarn fe storybook:deploy -- --ci --host-token-env-variable=GITHUB_TOKEN
```

Comment:
I assume this will deploy to `/docs` by default?

Reply:
To confirm, it builds into `/dist-storybook` which is then deployed to a `gh-pages` branch.